dids/didcore/document.go

```diff
@@ -194,6 +194,17 @@ func (d *Document) AddService(service *Service) {
 	d.Service = append(d.Service, service)
 }
 
+// GetAbsoluteResourceID returns a fully qualified ID for a document resource (e.g. service, verification method)
+// Document Resource IDs are allowed to be relative DID URLs as a means to reduce storage size of DID Documents.
+// More info here: https://www.w3.org/TR/did-core/#relative-did-urls
+func (d *Document) GetAbsoluteResourceID(id string) string {
+	if id[0] == '#' {
```

Comment:
I'd suggest using `strings.HasPrefix(id, "#")` for these kinds of checks, just in case an empty string is passed.

Reply:
Ok awesome. Will create an issue!